apollo-portal/pom.xml

```diff
@@ -31,6 +31,12 @@
 			<artifactId>h2</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.mockito</groupId>
+			<artifactId>mockito-core</artifactId>
+			<version>2.15.0</version>
```

Comment:
is there any new feature of mockito used?

Reply:
Yes, that's another question I would ask. Why do we stick to mockito 1.10? The `spring-starter-test` specify the version 2.x, and in this patch, I create some integration tests that need to skip the authorization, see [SkipAuthorizationConfiguration](https://github.com/ctripcorp/apollo/pull/1869/files#diff-5311c9021e81de1f6cfb141fd4f1218a) for detail. The changes are not just for this patch, it should be useful in the future when we need to add more integration tests. If I've missed anything, please correct me. Thanks!

Reply:
Clearly speaking, mockito 2.x allows us to mock beans that are to be injected into Spring container.

Reply:
Last time I tried to upgrade mocktio to 2.x, but met some errors in running all the tests

Reply:
I know that, I've tried too, and I finally managed to eliminate all but one exception in `com.ctrip.framework.apollo.configservice.service.AppNamespaceServiceWithCacheTest#testAppNamespace`, and I don't find a way to solve this eventually, that's why I only upgraded the mockito version in module `apollo-portal`

Reply:
got it, I will try to fix that later (if ever possible)

Reply:
the `com.ctrip.framework.apollo.configservice.service.AppNamespaceServiceWithCacheTest#testAppNamespace` case should be fixed in #1895